src/modules/mediaplayer/MpMprisInterface.cpp

```diff
@@ -597,6 +375,6 @@ MP_IMPLEMENT_DESCRIPTOR(
 MpClementineInterface::MpClementineInterface()
     : MpMprisInterface()
 {
-	m_szServiceName = "org.mpris.clementine";
+	m_szServiceName = "org.mpris.MediaPlayer2.clementine";
```

Comment:
I'm not familiar with this API (and too lazy to check), but shouldn't there be a way to check a generic media player, without listing all of possibilities? E.g. what KDE's lock screen or KDE's tray icon are doing

Currently at least `org.mpris.MediaPlayer2.strawberry` is missing from this list

Reply:
yes, we could enumerate all services named `org.mpris.MediaPlayer2.*` and use them, since they are supposed to expose the same exact dbus interfaces.
In the past we had to use different service named for each player, that's why we have a different class for each of them. Now we could just implement a "generic mpris" class and check the available players at runtime.

Reply:
The new commit adds 2 new interfaces. The first one is for strawberry, the second one is a generic fallback for any other player implementing a mpris2 interface.
At first i thought about removing the player-specific interfaces, but this would remove the ability to switch from one player to another from the options.